<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UAV2022Class</class>
 <widget class="QWidget" name="UAV2022Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>955</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UAV2022</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="6" column="1">
    <widget class="QPushButton" name="pBt_Locate">
     <property name="text">
      <string>矩形框和中心点标定</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="pBt_Diff">
     <property name="text">
      <string>帧间差分</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QPushButton" name="pBt_radius">
     <property name="text">
      <string>调整极坐标半径</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="11">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="pBt_LoadImage_2">
     <property name="text">
      <string>图片2</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="pBt_LoadImage">
     <property name="text">
      <string>图片1</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="UAV2022.qrc"/>
 </resources>
 <connections/>
</ui>
